tools/kprint/main.cpp

```diff
@@ -62,40 +102,142 @@ int main (int argc, char **argv) {
   comms.insert("\\or");
   std::map<std::string, std::vector<std::string>> colors;
 
-  // load information about definition written by k frontend
-  std::ifstream iformats(argv[1] + std::string("/format-att.txt"));
-  std::ifstream ihooks(argv[1] + std::string("/hook-att.txt"));
-  std::ifstream iassocs(argv[1] + std::string("/assoc-att.txt"));
-  std::ifstream icomms(argv[1] + std::string("/comm-att.txt"));
-  std::ifstream icolors(argv[1] + std::string("/color-att.txt"));
+  std::map<std::string, std::set<std::string>> priorities, leftAssoc, rightAssoc;
+  leftAssoc["kseq"].insert("kseq");
+  leftAssoc["append"].insert("append");
+  leftAssoc["\\and"].insert("\\and");
+  leftAssoc["\\or"].insert("\\or");
+  leftAssoc["\\rewrites"].insert("\\rewrites");
+  rightAssoc["\\rewrites"].insert("\\rewrites");
+  priorities["\\exists"].insert("\\implies");
```

Comment:
btw, unless we're need this to work with older compilers, you should be able to do
```
priorities["\\exists"] = {\\implies", "\\or", "\\and", "\\equals", "\\floor", ...}
```
(I'm not sure how readable a map initializer list would be, so I only suggested a set one).